openapi.yml

```diff
@@ -62,6 +62,68 @@ paths:
           description: FlowMod messages sent.
         '400':
            description: Invalid JSON flow.
+  '/api/kytos/flow_manager/v2/flows_by_switch':
+    post:
+      tags:
+        - Add
+      summary: Insert flows by switch.
+      parameters:
+        - name: force
```

Comment:
The force type we've been using is `boolean` is the expect since we don't expect other non binary states to be needed, so if you can follow the same pattern.

Reply:
Changed